### PR TITLE
docs: format sub-set of files to comply with template-2

### LINKED ISSUE #5: Update the Why files to align with the template
* update to new template
* go through for typos
* change to qmd?
* remove authors (#26) and make sure that dates stays as the modified date and not when the website was updated. See Decision #33

### PR DESCRIPTION
**Pull request by K-Beicher**
## Description

- This PR contains the minor edits to several why documents. These are:
Github Projects - an issue has been created to edit this further (#36)
Github - an issue has been created to write this up (#37)
Material Design
MIT License
Poetry
PolyRepo - an issue has been created to edit this further (#39)

## Related Issues

Closes One of many that is branched of a single issue

## Reviewer Focus

This PR only needs a quick review.

## Checklist

For general documentation:

- [x] Spell-check
    - [ ] US
    - [ ] UK
- [x] Did the page(s) preview correctly on your machine without breaking
- [x] New category words (keywords) (if any) added to the code snippet file

### CODE REVIEW

File: why-material-design.qmd [hunk 1]
```diff
@@ -31,45 +30,73 @@ We have considered the following:
...
-    - Has multiple implementations, which might take time to decide on.
+::: {.columns}
+::: {.column}
+#### Benefits
+
+- Has multiple implementations, like [Materialize Web](https://materializeweb.com/) and [BeerCSS](https://www.beercss.com/) that works well with Django
+- Prioritizes CSS over JavaScript
+- Very customizable
+- Looks very aesthetically pleasing (backed by the strong design community within Google)
+- [Figma](https://www.figma.com/) (used for sketching out the UI) has great support for Material Design
```
Comment (signekb) [inline why-material-design.qmd:60]:
```suggestion
- [Figma](https://www.figma.com/) (which can be used for sketching out the UI) has great support for Material Design
```

File: why-material-design.qmd [hunk 2]
```diff
@@ -31,45 +30,73 @@ We have considered the following:
...
-- **Pros**:
-    - CSS only.
-    - Very highly customisable.
-- **Cons**:
-    - Requires time and skill to customize
-    - Relies on [Node.js](https://nodejs.org/en) (via [npm](https://www.npmjs.com/)) which is an additional dependency and needs time to learn to use
+::: {.columns}
+::: {.column}
+#### Benefits
+- CSS only
```
Comment (signekb) [inline why-material-design.qmd:76]:
```suggestion
#### Benefits

- CSS only
```

File: why-material-design.qmd [hunk 3]
```diff
@@ -31,45 +30,73 @@ We have considered the following:
...
+- Very highly customisable
+:::
+::: {.column}
+#### Drawbacks
+- Requires time and skill to customize
```
Comment (signekb) [inline why-material-design.qmd:81]:
```suggestion
#### Drawbacks

- Requires time and skill to customize
```

File: why-material-design.qmd [hunk 4]
```diff
@@ -31,45 +30,73 @@ We have considered the following:
...
 
-- **Pros**:
-    - Seems relatively easy to use.
-- **Cons**:
-    - Requires Node.js (via npm) to install.
-
+::: {.columns}
+::: {.column}
+#### Benefits
+- Seems relatively easy to use
```
Comment (signekb) [inline why-material-design.qmd:93]:
```suggestion
#### Benefits

- Seems relatively easy to use
```

File: why-material-design.qmd [hunk 5]
```diff
@@ -31,45 +30,73 @@ We have considered the following:
...
+:::
+::: {.column}
+#### Drawbacks
+- Requires Node.js (via npm) to install
```
Comment (signekb) [inline why-material-design.qmd:97]:
```suggestion
#### Drawbacks

- Requires Node.js (via npm) to install
```

File: why-poetry.qmd [hunk 6]
```diff
@@ -11,95 +10,173 @@ categories:
...
-    - Installation and resolving package dependency trees (so there are no conflicts) can be a bit slow
-    - Not completely PEP compliant (not yet support [PEP 621](https://peps.python.org/pep-0621/), but an Issue on it is [here](https://github.com/python-poetry/roadmap/issues/3)), though this isn't a critical problem
+[Poetry](https://python-poetry.org/) was built and designed to address
+some of the short comings of pipenv and is specifically designed with
+package management and project development in mind.
+
+::: columns
+::: column
+#### Benefits
+
```
Comment (signekb) [inline why-poetry.qmd:96]:
```suggestion

```

File: why-poetry.qmd [hunk 7]
```diff
@@ -11,95 +10,173 @@ categories:
...
+- Similar dependency management to other languages
+- Installable with `pipx`
+- Integrates well with existing IDE/editors (PyCharm and VSCode)
+
+:::
+
+::: column
+#### Drawbacks
+
+
```
Comment (signekb) [inline why-poetry.qmd:112]:
```suggestion

```

**Review by signekb — CHANGES_REQUESTED** [review]
Nice! Just some minor suggestions :)

**Review by lwjohnst86 — APPROVED** [review]

**Pull request merged**